src/MoneyAdmin.Domain.Core/Commands/CommandResult.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace MoneyAdmin.Domain.Core.Commands
+{
+    public class CommandResult
+    {
+
+        public CommandResult(Exception exception) : this()
+        {
+            Exception = exception;
+        }
+
+        public CommandResult()
+        {
+        }
+        public Exception Exception { get; }
+
+        public bool IsSuccess => Exception == null;
+
+        public static CommandResult Success() => new CommandResult();
+        public Task<CommandResult> AsTask => Task.FromResult(this);
```

Comment:
Fix the correct elements order follow the [documentation](https://stackoverflow.com/questions/150479/order-of-items-in-classes-fields-properties-constructors-methods/310967#310967)